config/locales/members.nl.yml

```diff
@@ -26,6 +31,7 @@ nl:
       index:
         activities_calendar: Activiteitenkalender
         copy_ICS: Kopieer Webcal link
```

Comment:
The dutch translation doesn't include the "for all activities part".

Reply:
Fuck